src/nanoemoji/nanoemoji.py

```diff
@@ -222,6 +253,23 @@ def write_preamble(nw):
     nw.newline()
 
 
+@functools.lru_cache()
+def _chrome_command() -> str:
+    cmd, validator = {
+        "Linux": ("google-chrome", shutil.which),
+        "Darwin": (
+            "/Applications/Google Chrome.app/Contents/MacOS/Google Chrome",
+            lambda s: Path(s).is_file(),
+        ),
+        "Windows": ("chrome", shutil.which),
```

Comment:
have you actually tested this actually works on windows? Not a blocker from my POV if this (self-testing) feature only works on *nix